optax/losses/_classification.py

```diff
@@ -20,8 +20,14 @@
 import chex
 import jax
 import jax.numpy as jnp
+import numpy as np
 from optax import projections
 
+if np.__version__.startswith('1.'):
+  from numpy.core.numeric import normalize_axis_index, normalize_axis_tuple
```

Comment:
could we vendor a simplified version of `normalize_axis_index` so as to not depend on numpy here?